pkg/crypto/bls/bbs12381g2pub/bls.go

```diff
@@ -0,0 +1,227 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package bbs12381g2pub
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"hash"
+
+	bls12381 "github.com/kilic/bls12-381"
+	"github.com/phoreproject/bls"
+	"golang.org/x/crypto/blake2b"
+)
+
+// BlsG2Pub defines BBS+ signature scheme where public key is a point in the field of G2.
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Renamed to `BBSG2Pub`.